asgi_ipc.py

```diff
@@ -1,11 +1,11 @@
 from __future__ import unicode_literals
 
-import mmap
 import msgpack
+from pathlib import Path
```

Comment:
Pathlib is Python 3 only, and unfortunately we still have to support Python 2 here (I suspect this is why the Travis build failed)